### ISSUE TITLE
SE not printed with IRTpars

### ISSUE THREAD

**Issue by drabinova** [philchalmers/mirt#99]
Hi Phil,

I noticed that when I want to print estimated coefficients from my `mirt` model in IRT parameterization (i.e. with `IRTparam = T`), it is not possible to print also standard errors. Moreover, something weird happen to parameters `g` and `u`. I would expect that their values won't change.

Check my code:
```
data <- expand.table(LSAT7)
fit <- mirt(data, 1, SE = TRUE)
# with CI, everything seems OK
coef(fit)
# with SE, there is a logit of g and u parameters
coef(fit, printSE = T)
# no CI or SE
coef(fit, IRTpars = T)
# still no SE, again logit of g and u
coef(fit, IRTpars = T, printSE = T) 
```

I don't know if this behavior should be expected and is wanted or I've just missed something. Thanks for any suggestions!

**Comment by philchalmers** [comment]
This is expected behaviour. g and u are organised as logits internally, so the values represent this (hence, the standard errors would be in logit units as well).

**Comment by drabinova** [comment]
Thank you for quick reply. Now it is clear about logit. However, with IRT parameterization, SE or CI are not printed anymore. Is it possible to make `coef()` print them?

**Comment by philchalmers** [comment]
No, because they typically require the application of the delta method, which I'm not a fan of for these types of transformations. See this thread for more details: https://groups.google.com/forum/#!searchin/mirt-package/delta$20method%7Csort:relevance/mirt-package/qlHrupGOl2A/p9f_3y3SBQAJ

**Comment by drabinova** [comment]
Thank you for explanation and details!
